catalog/transformer/catalog-transformer-streaming-impl/src/main/java/org/codice/ddf/transformer/xml/streaming/impl/GmlHandler.java

```diff
@@ -82,7 +97,12 @@ public GmlHandler(GMLHandler gmlHandler, Gml3ToWkt gml3Converter) {
      */
     @Override
     public List<Attribute> getAttributes() {
-        return attributes;
+        return saxEventHandlerUtils.getCombinedMultiValuedAttributes(getSupportedAttributeDescriptors(), attributes);
+    }
+
+    @Override
+    public Set<AttributeDescriptor> getSupportedAttributeDescriptors() {
```

Comment:
You could probably add this as a method in the `AbstractSaxEventHandler`, then you would only have to implement this method if you don't want to `return the static set of attributeDescriptors`